<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessionManager</class>
 <widget class="QDialog" name="SessionManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Session Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/terminal.svg</normaloff>:/images/terminal.svg</iconset>
  </property>
  <layout class="QHBoxLayout" name="main_layout">
   <item>
    <widget class="QListWidget" name="m_session_list">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="m_layout_button_row">
    <layout class="QVBoxLayout" name="vertical_layout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="m_bt_switch">
        <property name="text">
         <string>&amp;Switch to</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_bt_clone">
        <property name="text">
         <string>C&amp;lone</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_bt_rename">
        <property name="text">
         <string>&amp;Rename</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_bt_delete">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="m_bt_close">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_bt_close</sender>
   <signal>clicked()</signal>
   <receiver>SessionManager</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
